packages/safe-tools-client/app/components/schedule-payment-form-action-card/index.gts

```diff
@@ -199,7 +197,7 @@ export default class SchedulePaymentFormActionCard extends Component<Signature>
             @placeholder="Choose a Gas Token"
             @value={{this.selectedGasToken}}
             @onChooseToken={{this.onSelectGasToken}}
-            @tokens={{this.gasTokens}}
+            @tokens={{withTokenIcons this.tokens.gasTokens.value}}
```

Comment:
With Polygon selected:

<img width="238" alt="Cardstack Safe Tools 2022-11-17 12-42-55" src="https://user-images.githubusercontent.com/43280/202531698-19a4dcba-7719-4f72-aba1-3b266002a9be.png">

I gather this is an SDK thing but I’m wondering if that’s expected. If so, is there better wording? (And a margin maybe)

Reply:
We shouldn't ever have a case of no gas tokens found once we properly configure everything.